RELEASE.md

```diff
@@ -108,21 +113,15 @@ Signing a tag with a GPG key is appreciated, but in case you can't add a GPG key
 
 Once a tag is created, the release process through CircleCI will be triggered for this tag and Circle CI will draft the GitHub release using the `prombot` account.
 
-Now all you can do is to wait for tarballs to be uploaded to the Github release and the container images to be pushed to the Docker Hub and Quay.io. Once that has happened, click _Publish release_, which will make the release publicly visible and create a GitHub notification.
+Finally, wait for the build step for the tag to finish. The point here is to wait for tarballs to be uploaded to the Github release and the container images to be pushed to the Docker Hub and Quay.io. Once that has happened, click _Publish release_, which will make the release publicly visible and create a GitHub notification.
+** Note: ** for a release candidate version the _This is a pre-release_ box when drafting the release in the Github UI.
+
+### 3. Wrapping up
 
-### Wrapping up
+For release candidate versions, run the benchmark for 3 days using the `/prombench vX.Y.Z` command, `vX.Y.Z` being the latest stable patch release's tag of the previous minor release series, such as `v2.15.2`
 
 If the release has happened in the latest release branch, merge the changes into master.
```

Comment:
We rarely create patch releases for previous minor releases, but if we do, we most likely want to merge it also back into master. (Technically, that's a change to the procedure, though, so we might want to keep it out of of this PR.)